CMakeLists.txt

```diff
@@ -1,13 +1,34 @@
-CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.13)
+set(CMAKE_CONFIGURATION_TYPES "Release;RelWithDebInfo;Debug" CACHE STRING "Build type selections" FORCE)
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE RelWithDebInfo CACHE STRING "default build type")
+endif()
+project(kissnet_ci
+  LANGUAGES CXX
+  DESCRIPTION "header-only C++ network interface"
+  HOMEPAGE_URL https://github.com/Ybalrid/kissnet
+  VERSION 0.2.0)
```

Comment:
I don't think we should put it out as version 0.2 just yet 😉 

Keep that at version 0.1, we bump to 0.2 when we will break user-facing API